Week-4/assignment3.js

```diff
@@ -0,0 +1,26 @@
+let obj = {
+  [Symbol.iterator]() {
```

Comment:
@relhanakshay can you change this implementation slightly to accept 'n' value which will be used to stop the loop. Try to generate 'n' fibonacci series.

I think this is not accepting any argument and max it will iterate for max 7 or 8 i suppose. Correct me if i am wrong